Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -27,6 +27,8 @@
 //
 // Version: 1.0.12
 
+#if !ENABLE_FACEPUNCH_STEAM
```

Comment:
I think it would be better to integrate this conditional compile into the one on line 32. Otherwise I think the 'fake' steamwork.net code won't work in this file, correct?